indigo_app/templates/indigo_api/work_new_batch.html

```diff
@@ -108,12 +105,12 @@ <h4 class="card-header">
               <th>Number</th>
               <th>Year</th>
               <th>FRBR URI</th>
+              <th>Taxonomy</th>
               <th>Tasks</th>
               <th>Relationships</th>
-              <th>Actions</th>
+              <th>Notes (shown in Preview only)</th>
```

Comment:
Minor: I don't think anyone's going to be confused if this is just shown in preview, but not after import. So perhaps the heading can just be "Notes"/"Comments"/"Import notes".